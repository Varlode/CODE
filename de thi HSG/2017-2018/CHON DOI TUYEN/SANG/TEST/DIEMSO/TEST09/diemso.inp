1000
191298 869512 752315 243017 424427 832201 431941 546314 402503 785569 117427 650105 194600 716449 159005 214641 216810 427062 931098 820286 839568 917218 995748 369691 706413 174979 144241 687219 555515 831820 667762 662209 572122 174579 15771 582730 530255 763723 161911 750108 62944 217825 853699 933969 829905 141978 735049 799105 799540 763439 877687 513030 185705 535707 413172 729008 686365 565103 98111 727198 566161 931618 343795 211141 777794 907404 552104 244623 560780 685856 87768 168939 648155 701612 839943 26788 830753 649100 414323 272483 61146 813539 155937 734475 363971 458835 979420 385455 654103 495448 784892 156567 304511 554194 958610 298510 806308 310765 882812 593214 433045 52507 853866 40275 974770 360201 874121 371105 312360 389921 682743 307025 80515 10128 183627 426791 226890 962095 372853 753850 834157 108285 126295 89052 485878 372513 575892 227502 316721 528234 117725 773308 390449 166055 480619 311508 399100 794185 189769 202799 609524 383789 686895 50112 480361 998005 723751 379090 182473 339694 494710 4635 995012 30004 876053 935527 459165 253819 562946 667252 543129 772481 727092 242077 523992 356386 584434 659746 860450 636220 547056 675771 370919 359389 727595 79571 54684 518299 779942 68133 741865 556162 39916 316010 145693 871437 419479 408749 991731 518121 355530 543218 78942 167613 299610 448301 566580 542940 830814 927802 489868 806858 784283 734731 814394 970807 584180 115354 595943 635241 235568 548192 650843 785683 866813 320293 690053 447326 62899 813074 481505 714905 537657 715906 327489 481659 727388 83514 591437 240132 617946 194997 763132 416768 727745 165224 837250 732697 222326 99736 694250 241450 716605 927264 694359 81192 878616 965074 594203 134188 791498 612547 802769 209040 295303 843647 846464 348881 629348 814161 790525 665342 668512 296949 497471 321441 157850 315483 775957 910528 139979 779960 887506 176899 735095 286318 408660 931837 593650 686059 312784 507627 108633 480293 221884 263395 745907 274355 338998 462148 772601 926170 726131 120423 617211 74044 547829 943588 859317 870360 733153 964555 472816 466222 425987 226138 119516 509777 902804 563622 192316 736678 761687 157137 960164 909455 330351 709347 570138 769127 339511 446384 82695 949873 457875 77279 136344 48137 506333 592629 882291 936962 191115 872469 743750 990462 206499 807948 19474 685404 609718 337228 90959 812465 913013 400973 584435 131293 333210 5177 402035 387381 127506 649097 793818 584564 69952 960696 571422 689628 461593 847201 488864 814840 387210 311583 384474 811961 226722 124057 511834 260205 565748 697147 152719 400712 919286 276665 681674 917164 346610 448478 973162 360541 526229 584617 351128 807009 182575 314372 925104 867317 200187 935393 752068 59224 836325 308680 710715 34692 618671 950502 616505 820653 410070 723269 378291 154748 60068 184895 978676 782638 926246 766912 202987 523708 384714 534848 948008 991409 345522 1735 141304 452370 620913 294338 226401 279636 674375 386701 170708 414761 278993 996358 467661 447349 47563 638186 125273 477166 679586 33246 919640 840881 340145 199303 997837 875563 400140 499385 954835 709788 559392 193228 360265 635301 879763 77072 593776 747771 738822 83026 124120 320526 504162 153330 884496 547165 429899 966095 978092 564636 900358 210709 493293 908159 126546 27909 62729 785051 641395 295902 52857 221826 713031 761657 770281 494840 575445 526846 487801 363976 301132 827240 39637 754428 519037 603844 967323 326537 78440 354266 327318 4270 684710 324986 494509 254979 840625 956225 180096 885281 701398 297912 966909 112575 451337 732361 459154 271085 890262 342156 702555 519524 874870 597974 998186 468493 945765 434217 226774 601925 366167 997760 12398 887254 962763 596496 292613 571289 806807 557492 487990 277626 892040 610284 824509 719107 378699 918782 839864 876869 269790 888870 53026 152356 416366 3854 962890 430641 851549 227481 905839 323950 898279 158185 792013 852286 186767 283318 717678 256016 97532 797759 253202 685039 842134 585373 119290 358718 695701 797682 486698 909012 75021 462166 475514 314041 27229 146432 714716 18718 797816 853040 575430 599902 332400 295580 90639 983180 625507 37418 453513 77738 917455 860373 760940 401255 147187 256599 56960 474909 881291 883195 471631 381885 568644 565480 536084 11342 125491 107017 456867 674589 504512 937080 984480 844044 744696 243525 666812 911417 756717 21829 467499 33695 327142 511856 593101 882900 807974 659527 862002 309118 626106 775682 577350 727987 6610 274846 138645 590180 787362 712683 186893 777555 621440 999577 942042 551367 180748 391947 216147 579354 164575 9139 375628 297526 164603 807578 762180 155696 429689 218683 115668 833037 388603 782019 571585 90731 736558 367180 68191 132551 594621 165277 756462 265780 719700 257205 189196 473678 494639 967889 481301 956637 930217 990920 917523 764257 160608 707805 967492 346972 356525 297802 190196 594566 263420 437441 340920 477189 181039 570100 478456 805459 216574 466138 489574 432189 294522 178485 845908 965366 868411 676885 27067 522645 357710 922737 322918 299543 390542 955058 724242 615812 122171 967716 457385 188460 880209 806371 359516 444049 33710 359239 226907 426455 260800 362314 256536 263962 179337 555813 762791 7458 466868 271480 445159 606976 979554 28274 930608 425441 345215 697852 291191 781347 155377 597586 960075 668642 536626 25219 383339 585783 826623 966504 176124 944158 226612 178250 641227 923159 33825 329395 432782 136663 916064 268284 545773 761674 959251 317987 179025 59842 551377 764478 368668 656782 12435 13156 176327 53747 79342 339839 731641 190379 719661 219893 34795 928133 410705 705375 576206 56690 96591 706662 91065 155941 646708 248350 672440 362036 774449 576784 137805 177353 399020 699853 730600 473274 426621 866800 291540 401289 705869 235227 71813 326042 264172 551934 203754 296189 568465 221480 743497 255204 711069 378075 198239 845176 514130 631511 6841 2985 770203 354840 945382 413554 86593 755086 274568 521028 569707 454862 249213 697743 585815 886526 706318 17396 602633 827425 981338 483903 272110 198661 347334 153820 565833 381161 96943 797067 896578 679938 529073 903411 994395 643229 937604 948852 275505 352481 180972 501597 518056 924468 616242 450054 817268 387097 650975 573240 717764 894752 644704 514188 895698 669113 224459 82446 598947 479226 828124 895930 25185 241064 829088 89600 696702 565958 177179 391739 189547 838221 739760 485454 655220 878444 634768 149679 739187 616776 994468 445234 216387 953788 130185 677232 950991 627918 50714 51740 13836 518168 328538 208393 133855 591125 401997 104035 789891 325782 41956 62606 614601 917755 9754 135240 931953 986883 738560 891420 764466 526916 399886 916108 376937 493627 986300 717853 616632 543668 876539 139857 204046 83455 637325 285269 79622 117252 951944 210282 270757 406001 828402 661518 877998 335151 433742 372552 40584 744158 278571 268004 373466 790965 304597 315407 
3
27 162
90 146
62 130
