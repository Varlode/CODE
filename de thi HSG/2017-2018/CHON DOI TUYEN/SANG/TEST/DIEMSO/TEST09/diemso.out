67329517
25513251
32940908
